src/main/java/ui/ResultView.java

```diff
@@ -0,0 +1,40 @@
+package ui;
+
+import common.StringUtils;
+import domain.*;
+
+public class ResultView {
+    private static final String VERTICAL = "|";
+    private static final String HORIZONTAL = "-";
+    private static final String BLANK = " ";
+    private static final String NEWLINE = "\n";
+    private static final int NUMBER_OF_TEMPLATE_BLANK = 4;
+    private static final int NUMBER_OF_TEMPLATE_HORIZONTAL = 5;
+    private static final int NUMBER_OF_NAME_PADDING = 5;
+
+    public static void print(final LadderGame ladderGame) {
+        printPlayerNames(ladderGame.getPlayerNames());
+        printLadder(ladderGame.getLadder());
+    }
+
+    private static void printPlayerNames(final PlayerNames playerNames) {
+        playerNames.stream().forEach(s -> {
+            System.out.print(StringUtils.padByBlank(s, NUMBER_OF_NAME_PADDING).concat(BLANK));
+        });
+        System.out.print(NEWLINE);
+    }
+
+    public static void printLadder(final Ladder ladder) {
+        ladder.getLines().stream().map(Line::getPoints).forEach(points -> {
+            System.out.print(StringUtils.copyAndJoin(NUMBER_OF_TEMPLATE_BLANK, BLANK));
+            points.stream().forEach( point -> System.out.print(convertPointToString(point.isPresent())) );
+            System.out.println(VERTICAL);
+        });
```

Comment:
```suggestion
        ladder.getLines().stream()
        .map(Line::getPoints)
        .forEach(points -> {
            System.out.print(StringUtils.copyAndJoin(NUMBER_OF_TEMPLATE_BLANK, BLANK));
            points.stream().forEach( point -> System.out.print(convertPointToString(point.isPresent())) );
            System.out.println(VERTICAL);
        });
```
스트림은 기능단위로 줄바꿈하여 사용하는것을 권장 드려요. 
또한 이와같이 스트림 내부에 코드가 많아지는 경우 스트림을 사용하지 않고 구현하는것을 권장합니다 😄 
이번 과정에서 람다, 스트림의 사용이 필수는 아닙니다 !